File: 初级算法/23_8_4/2.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ans = []
        for i in range(n):
            j = i + 1
            if j % 3 == 0 and j % 5 == 0 :
                ans.append("FizzBuzz")
            elif j % 3 == 0:
                ans.append("Fizz")
            elif j % 5 == 0:
                ans.append("Buzz")
            else:
                ans.append(str(j))

        return ans

File: weekGames/weekGame_361/2.cpp
class Solution {
public:
    int minimumOperations(string num) {
        int n = num.size(), i = n - 1, ans = -1;
        int f[2] = {0, 0}; // 0 or 5
        while(i >= 0)
        {
            if(num[i] == '0')
            {
                if(f[0]) return ans;
                f[0] = 1;
            }
            else if(num[i] == '5')
            {
                if(f[0]) return ans;
                f[1] = 1;
            }
            else if(f[1] && (num[i] == '2' || num[i] == '7'))
            {
                return ans;
            }
            ans++, i--;
        }
        return n - f[0];
    }
};